App/HostedReactor.fs

```diff
@@ -0,0 +1,67 @@
+module HostedReactor
+
+open System
+open Microsoft.Extensions.Hosting
+open Propulsion.Internal
+open Propulsion.MessageDb
+
+type HostedReactor(connectionString, log, service: InvoiceNumberingReactor.Service, checkpointService: CheckpointStore.CheckpointService) =
+  inherit BackgroundService()
+
+  let stats =
+    { new Propulsion.Streams.Stats<_>(log, TimeSpan.FromMinutes 1, TimeSpan.FromMinutes 1) with
+        member _.HandleOk x = ()
+        member _.HandleExn(log, x) = () }
+
+  [<Literal>]
+  let groupName = "InvoiceNumberingReactor"
+
+  let categories = [| Invoice.Category |]
+
+  let checkpoints = CheckpointStore.CheckpointStore(checkpointService, groupName, TimeSpan.FromSeconds 10)
+  let maxReadAhead = 100
+  let maxConcurrentStreams = 2
+
+  let handle struct(stream, events) =
+    async {
+      for event in events do
+        do! service.Handle(stream, event)
```

Comment:
in a parallel universe I use one char tabs!

Reply:
We can meet in the middle and use 3?

Reply:
We could also submit to `0x9` already

Reply:
I've always wanted to try `5` one day ;) The day Fantomas applies `0x9`, we can do that!

Reply:
The man is wise ;)
(Does go or any other `fmt` use tabs?)

Reply:
I'm not aware of any language that defaults to tabs. Hopefully that changes soon 😉, when it happens you can have your 4 indents, and I can have my 2

Reply:
While I got used to 4 in Turbo/Borland Pascal, C and C++, I suspect I'd actually run it with 2 at that point.
An indent of 2 in F# is triggering for me as I strongly associate it with old pain of people wanting to write Haskell and/or Arrows-based stuff in F#.